server/grpc/service.go

```diff
@@ -3,7 +3,9 @@ package grpc
 import (
 	"context"
 
+	protoApi "github.com/mesg-foundation/core/protobuf/api"
```

Comment:
do not use camelCase in imports.

Reply:
Actually this import is useless. It was already imported as `protobuf_api` 😂